scripts/builder.js

```diff
@@ -0,0 +1,59 @@
+var articles = [];
+
+function Article (options) {
+  this.title = options.title;
+  this.category = options.category;
+  this.abstract = options.abstract;
+  this.coders = options.coders;
+  this.repoUrl = options.repoUrl;
+  this.dateStamp = options.dateStamp;
+};
+
```

Comment:
Well done with your article constructor! Was there a preference between writing it out this way instead of using the "for" loop using "key" and "opts"?